components/plugin_marketplace/marketplace_modal.tsx

```diff
@@ -84,81 +90,89 @@ export const InstalledPlugins = ({installedPlugins, changeTab}) => {
     return <MarketplaceList plugins={installedPlugins}/>;
 };
 
-InstalledPlugins.propTypes = {
-    installedPlugins: PropTypes.array.isRequired,
-    changeTab: PropTypes.func,
+export type MarketplaceModalProps = {
+    show: boolean;
+    plugins: MarketplacePlugin[];
+    installedPlugins: MarketplacePlugin[];
+    siteURL: string;
+    pluginStatuses?: Dictionary<PluginStatusRedux>,
+    actions: {
+        closeModal: () => void;
+        fetchPlugins: (localOnly?: boolean) => Promise<{error?: Error}>;
+        filterPlugins(filter: string): Promise<{error?: Error}>;
+    };
+};
+
+type MarketplaceModalState = {
+    tabKey: any;
```

Comment:
Don't we have a better type for this?

Reply:
Do you think it's worth to `//eslint-disable` the `any` warning?

Reply:
Make the code more clear in ff96cc11796f94448d1f7d774fc3094da9a4b88f